flight/Modules/UAVOMSPBridge/UAVOMSPBridge.c

```diff
@@ -321,12 +190,99 @@ static void msp_send_ident(struct msp_bridge *m)
 
 static void msp_send_raw_gps(struct msp_bridge *m)
 {
-	// TODO
+	union {
+		uint8_t buf[0];
+		struct {
+			uint8_t  fix;                 // 0 or 1
+			uint8_t  num_sat;
+			int32_t lat;                  // 1 / 10 000 000 deg
```

Comment:
I see:
```
uint8_t  fix;
uint8_t  num_sat;
int32_t lat; 
int32_t lon; 
```

Where there are two spaces for `fix` and `num_sat`, but one for everything else.